src/pages/journal/journal.module.ts

```diff
@@ -48,6 +49,7 @@ import { JournalLogsEffects } from './journal.logs.effects';
       JournalAnalyticsEffects,
       JournalLogsEffects,
     ]),
+    CandidateModule,
```

Comment:
Can't spot this being used in any of the Journal Changes

Reply:
As per our chat, this is required to bootstrap the candidate state module prior to navigating to the Waiting Room page. Moved up to `app.module.ts` as we said.